.github/workflows/github-action-test.yml

```
@@ -0,0 +1,17 @@
+name: Test github action
+
+on: [push]
+
+jobs:
+  action:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Download pluto
+        uses: ./github-action
+
+      - name: Pluto exists?
+        run: |
+          pluto --help
```

Usually I prefer to use the version command. That way when we review the test run we know which version got pulled. (Makes debugging and verifying easier)

```suggestion
          pluto version
```